seametrics/tracking/hota.py

```
@@ -0,0 +1,207 @@
+    matches is a list of (gt_idx, pred_idx, iou).
+    Pairs whose IoU is below threshold are discarded.
+    """
+    M, N = iou_mat.shape
+    if M == 0 or N == 0:
+        return [], list(range(M)), list(range(N))
+
+    row_ind, col_ind = linear_sum_assignment(-iou_mat)
+
+    matched_gt, matched_pr = set(), set()
+    matches = []
+    for r, c in zip(row_ind, col_ind):
```

Ruff (0.15.12)

[warning] `zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)